sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -107,6 +108,26 @@ def stop(self, timeout=1):
     if os.path.exists(self._staging_dir) and self._cleanup_staging_dir:
       shutil.rmtree(self._staging_dir, ignore_errors=True)
 
+  def GetJobMetrics(self, request, context=None):
+    if request.job_id not in self._jobs:
+      raise LookupError("Job {} does not exist".format(request.job_id))
+
+    result = self._jobs[request.job_id].result
+    monitoring_info_list = []
+    for mi in result._monitoring_infos_by_stage.values():
+      monitoring_info_list.extend(mi)
+
+    # Filter out system metrics
+    user_monitoring_info_list = [
+        x for x in monitoring_info_list
+        if monitoring_infos._is_user_monitoring_info(x) or
+        monitoring_infos._is_user_distribution_monitoring_info(x)
+    ]
+
+    return beam_job_api_pb2.GetJobMetricsResponse(
```

Comment:
FYI, here is some background. I was not aware that these existed in proto form now. Creating a proto was suggested but not pursued at the time.
https://s.apache.org/get-metrics-api

When I last worked here MetricResult did not have a proto format. The plan was to use MonitoringInfos for a language agnostic format, and then MetricResult would be a language specific format (python, java, go, etc.) Each runner should provide a way to return MonitoringInfos, and each language would have  a library to convert MonitoringInfos to MetricResult protos

Its hard for me to review, I don't think I am up to date on whatever the current plan/usage of these protos are.